azurepipelines/e2e_test/scenarios/test_runner/vm_setup/setup.sh

```diff
@@ -177,11 +255,10 @@ function test_shutdown_service() {
     done
 }
 
-configure_apt_repository
+configure_apt_repository "$distro" "$architecture"
```

Comment:
IS this on the latest commit from develop? It looks like it's missing my AIS SAS work (same as the above runners).